index.bs

```diff
@@ -542,6 +542,19 @@ url: https://w3c.github.io/permissions/#permissions-task-source; type: dfn;
 			* Custom format [=string/starts with=] `"web "`("web" followed by U+0020 SPACE) prefix
 				and suffix (after stripping out `"web "`) passes the [=parsing a MIME type=] check.
 
+	<h3 id="unsanitized-data-types-x"><dfn>Unsanitized data types</dfn></h3>
+
+		The implementation MAY recognize the native OS clipboard format description
+		for the following data types, to be able to populate the
```

Comment:
What do you mean by "correct description for paste events"?

Reply:
hm I guess we don't expose any MIME type that isn't supported by DataTransfer APIs or the default paste event, so I think we can remove it.